[2014-07-03 09:21:36] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Call to undefined method Illuminate\Routing\UrlGenerator::to_route()' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php:206 Stack trace: #0 [internal function]: Illuminate\Exception\Handler->handleShutdown() #1 {main} [] []
[2014-07-03 09:21:44] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #5 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #6 {main} [] []
[2014-07-03 09:21:44] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #5 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #6 {main} [] []
[2014-07-03 09:21:46] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Call to undefined method Illuminate\Routing\UrlGenerator::to_route()' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php:206 Stack trace: #0 [internal function]: Illuminate\Exception\Handler->handleShutdown() #1 {main} [] []
[2014-07-03 09:22:59] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Call to undefined method Illuminate\Routing\UrlGenerator::to_route()' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php:206 Stack trace: #0 [internal function]: Illuminate\Exception\Handler->handleShutdown() #1 {main} [] []
[2014-07-03 09:23:01] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #5 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #6 {main} [] []
[2014-07-03 09:23:01] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #5 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #6 {main} [] []
[2014-07-03 09:23:02] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Call to undefined method Illuminate\Routing\UrlGenerator::to_route()' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php:206 Stack trace: #0 [internal function]: Illuminate\Exception\Handler->handleShutdown() #1 {main} [] []
[2014-07-03 09:23:07] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #5 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #6 {main} [] []
[2014-07-03 09:23:12] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Call to undefined method Illuminate\Routing\UrlGenerator::to_route()' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php:206 Stack trace: #0 [internal function]: Illuminate\Exception\Handler->handleShutdown() #1 {main} [] []
[2014-07-03 09:23:51] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Call to undefined method Illuminate\Routing\UrlGenerator::to_route()' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php:206 Stack trace: #0 [internal function]: Illuminate\Exception\Handler->handleShutdown() #1 {main} [] []
[2014-07-03 09:25:17] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'syntax error, unexpected ';'' in /opt/lampp/htdocs/noyawa/app/storage/views/da0603b62b2d8cea6264c710a95731ac:32 Stack trace: #0 [internal function]: Illuminate\Exception\Handler->handleShutdown() #1 {main} [] []
[2014-07-03 09:25:34] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Call to undefined method Illuminate\Routing\UrlGenerator::to_action()' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php:206 Stack trace: #0 [internal function]: Illuminate\Exception\Handler->handleShutdown() #1 {main} [] []
[2014-07-03 09:29:48] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #5 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #6 {main} [] []
[2014-07-03 09:30:39] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Class 'Illuminate\Database\Eloquent' not found' in /opt/lampp/htdocs/noyawa/app/models/ExcelUpload.php:14 Stack trace: #0 [internal function]: Illuminate\Exception\Handler->handleShutdown() #1 {main} [] []
[2014-07-03 09:31:07] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #5 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #6 {main} [] []
[2014-07-03 09:36:22] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #5 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #6 {main} [] []
[2014-07-03 09:41:57] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #5 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #6 {main} [] []
[2014-07-03 09:42:44] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #5 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #6 {main} [] []
[2014-07-03 09:43:26] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #5 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #6 {main} [] []
[2014-07-03 09:43:35] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #5 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #6 {main} [] []
[2014-07-03 09:43:35] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #5 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #6 {main} [] []
[2014-07-03 09:43:36] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #5 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #6 {main} [] []
[2014-07-03 09:47:56] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #5 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #6 {main} [] []
[2014-07-03 09:47:58] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #5 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #6 {main} [] []
[2014-07-03 09:47:59] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #5 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #6 {main} [] []
[2014-07-03 09:59:45] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #5 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #6 {main} [] []
[2014-07-03 09:59:45] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #5 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #6 {main} [] []
[2014-07-03 10:01:07] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #5 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #6 {main} [] []
[2014-07-03 10:01:08] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #5 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #6 {main} [] []
[2014-07-03 10:30:12] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #5 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #6 {main} [] []
[2014-07-03 10:38:54] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #5 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #6 {main} [] []
[2014-07-03 10:39:29] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #5 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #6 {main} [] []
[2014-07-03 10:44:35] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #5 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #6 {main} [] []
[2014-07-03 10:45:32] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #5 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #6 {main} [] []
[2014-07-03 10:47:07] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #5 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #6 {main} [] []
[2014-07-03 10:47:23] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'syntax error, unexpected ''); ?>"></script>' (T_CONSTANT_ENCAPSED_STRING), expecting ',' or ';'' in /opt/lampp/htdocs/noyawa/app/storage/views/03e0dbe4c36ddd6ce612c74c9334d4d8:111 Stack trace: #0 [internal function]: Illuminate\Exception\Handler->handleShutdown() #1 {main} [] []
[2014-07-03 10:47:36] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #5 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #6 {main} [] []
[2014-07-03 10:49:53] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #5 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #6 {main} [] []
[2014-07-03 10:51:06] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #5 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #6 {main} [] []
[2014-07-03 10:51:26] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #5 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #6 {main} [] []
[2014-07-03 10:52:31] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #5 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #6 {main} [] []
[2014-07-03 10:52:45] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #5 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #6 {main} [] []
[2014-07-03 10:53:07] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #5 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #6 {main} [] []
[2014-07-03 10:54:01] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #5 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #6 {main} [] []
[2014-07-03 11:07:06] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #5 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #6 {main} [] []
[2014-07-03 11:09:51] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #5 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #6 {main} [] []
[2014-07-03 11:13:29] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #5 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #6 {main} [] []
[2014-07-03 11:14:12] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #5 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #6 {main} [] []
[2014-07-03 11:19:55] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #5 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #6 {main} [] []
[2014-07-03 11:22:54] log.INFO: Uploading excel file -> tools.sdf [] []
[2014-07-03 11:22:54] log.INFO: File uploaded -> tools.sdf :: Processing started... [] []
[2014-07-03 11:22:54] log.INFO: File is readable -> tools.sdf [] []
[2014-07-03 11:22:55] log.INFO: File has 1 records [] []
[2014-07-03 11:22:55] log.INFO: Processing of file completed [] []
[2014-07-03 11:24:14] log.INFO: Uploading excel file -> FreeToolsInstall.txt [] []
[2014-07-03 11:24:14] log.INFO: File uploaded -> FreeToolsInstall.txt :: Processing started... [] []
[2014-07-03 11:24:14] log.INFO: File is readable -> FreeToolsInstall.txt [] []
[2014-07-03 11:24:14] log.ERROR: exception 'ErrorException' with message 'DOMDocument::loadHTMLFile(): Tag null invalid in uploads/FreeToolsInstall.txt, line: 10' in /opt/lampp/htdocs/noyawa/vendor/phpoffice/phpexcel/Classes/PHPExcel/Reader/HTML.php:427 Stack trace: #0 [internal function]: Illuminate\Exception\Handler->handleError(2, 'DOMDocument::lo...', '/opt/lampp/htdo...', 427, Array) #1 /opt/lampp/htdocs/noyawa/vendor/phpoffice/phpexcel/Classes/PHPExcel/Reader/HTML.php(427): DOMDocument->loadHTMLFile('uploads/FreeToo...', 12) #2 /opt/lampp/htdocs/noyawa/vendor/phpoffice/phpexcel/Classes/PHPExcel/Reader/HTML.php(141): PHPExcel_Reader_HTML->loadIntoExisting('uploads/FreeToo...', Object(PHPExcel)) #3 /opt/lampp/htdocs/noyawa/vendor/phpoffice/phpexcel/Classes/PHPExcel/IOFactory.php(192): PHPExcel_Reader_HTML->load('uploads/FreeToo...') #4 /opt/lampp/htdocs/noyawa/app/controllers/UploadController.php(47): PHPExcel_IOFactory::load('uploads/FreeToo...') #5 [internal function]: UploadController->uploadExcel() #6 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #7 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('uploadExcel', Array) #8 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'uploadExcel', Array) #9 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #10 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #11 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #12 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #13 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #14 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #15 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #16 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #17 {main} [] []
[2014-07-03 11:29:58] log.INFO: Accessing postClient @ ClientController [] []
[2014-07-03 11:53:47] log.INFO: Accessing postClient @ ClientController [] []
[2014-07-03 11:54:12] log.INFO: Accessing postClient @ ClientController [] []
[2014-07-03 11:55:11] log.INFO: Accessing postClient @ ClientController [] []
[2014-07-03 11:56:08] log.INFO: Accessing postClient @ ClientController [] []
[2014-07-03 12:26:27] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #5 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #6 {main} [] []
[2014-07-03 12:26:35] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #5 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #6 {main} [] []
[2014-07-03 12:27:05] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #5 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #6 {main} [] []
[2014-07-03 12:27:09] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #5 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #6 {main} [] []
[2014-07-03 12:27:26] log.ERROR: exception 'PDOException' with message 'SQLSTATE[42S22]: Column not found: 1054 Unknown column 'client_educational_level' in 'field list'' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Database/Connection.php:272 Stack trace: #0 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Database/Connection.php(272): PDO->prepare('select `id`, `c...') #1 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Database/Connection.php(521): Illuminate\Database\Connection->Illuminate\Database\{closure}(Object(Illuminate\Database\MySqlConnection), 'select `id`, `c...', Array) #2 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Database/Connection.php(277): Illuminate\Database\Connection->run('select `id`, `c...', Array, Object(Closure)) #3 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php(1028): Illuminate\Database\Connection->select('select `id`, `c...', Array) #4 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php(1018): Illuminate\Database\Query\Builder->runSelect() #5 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php(1005): Illuminate\Database\Query\Builder->getFresh(Array) #6 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Builder.php(441): Illuminate\Database\Query\Builder->get(Array) #7 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Builder.php(132): Illuminate\Database\Eloquent\Builder->getModels(Array) #8 /opt/lampp/htdocs/noyawa/vendor/bllim/datatables/src/Bllim/Datatables/Datatables.php(84): Illuminate\Database\Eloquent\Builder->get() #9 /opt/lampp/htdocs/noyawa/vendor/bllim/datatables/src/Bllim/Datatables/Datatables.php(68): Bllim\Datatables\Datatables->get_result() #10 /opt/lampp/htdocs/noyawa/app/controllers/ClientController.php(111): Bllim\Datatables\Datatables->make() #11 [internal function]: ClientController->getData() #12 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #13 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getData', Array) #14 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getData', Array) #15 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #16 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #17 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #18 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #19 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #20 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #21 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #22 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #23 {main} Next exception 'Exception' with message 'SQLSTATE[42S22]: Column not found: 1054 Unknown column 'client_educational_level' in 'field list' (SQL: select `id`, `client_number`, `client_gender`, `client_age`, `client_educational_level`, `status`, `created_at` from `clients_sms_registration`) (Bindings: array ( ))' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Database/Connection.php:556 Stack trace: #0 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Database/Connection.php(529): Illuminate\Database\Connection->handleQueryException(Object(PDOException), 'select `id`, `c...', Array) #1 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Database/Connection.php(277): Illuminate\Database\Connection->run('select `id`, `c...', Array, Object(Closure)) #2 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php(1028): Illuminate\Database\Connection->select('select `id`, `c...', Array) #3 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php(1018): Illuminate\Database\Query\Builder->runSelect() #4 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php(1005): Illuminate\Database\Query\Builder->getFresh(Array) #5 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Builder.php(441): Illuminate\Database\Query\Builder->get(Array) #6 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Builder.php(132): Illuminate\Database\Eloquent\Builder->getModels(Array) #7 /opt/lampp/htdocs/noyawa/vendor/bllim/datatables/src/Bllim/Datatables/Datatables.php(84): Illuminate\Database\Eloquent\Builder->get() #8 /opt/lampp/htdocs/noyawa/vendor/bllim/datatables/src/Bllim/Datatables/Datatables.php(68): Bllim\Datatables\Datatables->get_result() #9 /opt/lampp/htdocs/noyawa/app/controllers/ClientController.php(111): Bllim\Datatables\Datatables->make() #10 [internal function]: ClientController->getData() #11 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #12 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getData', Array) #13 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getData', Array) #14 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #15 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #16 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #17 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #18 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #19 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #20 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #21 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #22 {main} [] []
[2014-07-03 12:27:57] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #5 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #6 {main} [] []
[2014-07-03 12:31:02] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #5 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #6 {main} [] []
[2014-07-03 12:31:06] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #5 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #6 {main} [] []
[2014-07-03 12:32:06] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #5 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #6 {main} [] []
[2014-07-03 12:33:09] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #5 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #6 {main} [] []
[2014-07-03 12:33:31] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #5 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #6 {main} [] []
[2014-07-03 12:34:13] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #5 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #6 {main} [] []
[2014-07-03 12:38:24] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #5 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #6 {main} [] []
